apps/aecore/src/aec_peer_connection.erl

```diff
@@ -37,27 +46,19 @@
 -export([init/1, handle_call/3, handle_cast/2, handle_info/2,
```

Comment:
I see above:
```
-define(FIRST_PING_TIMEOUT, 30000).
-define(NOISE_HS_TIMEOUT, 5000).
```
shall they be made configurable?

Reply:
Same here. I understand it could be configurable, but it didn't seem part of the ticket.